manifests/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: objectstorage-controller
+  namespace: objectstorage-system
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  selector:
+    matchLabels:
+      app: objectstorage-controller
+  template:
+    metadata:
+      labels:
+        app: objectstorage-controller
+    spec:
+      serviceAccountName: objectstorage-controller-sa
+      containers:
+        - name: objectstorage-controller
+          image: quay.io/containerobjectstorage/objectstorage-controller:latest
```

Comment:
Not sure if we hardcode image tags 'objectstorage-controller:latest'

Reply:
That is just the default. It can be overriden in the kustomize.yaml. We can update the image source and tag once we have the pipeline to build and publish images in place in official repos.

Reply:
SGTM, I was hoping that we declare variable for image tag so that it can be patched.

Reply:
We can (and for now IMO should) have `latest` here, since that's +- what you should get when deploying from a `main` Git branch.

As noted above, the overrides in `kustomize.yaml` should likely be removed (for now, at least). One *can* create an _overlay_ today to replace `latest` by whatever already today. Then, once there are releases of this project, for a given release, the correct tag can be set in this file (and, again, overridden using an _overlay_ if desired).

Reply:
Captured this in issue https://github.com/kubernetes-sigs/container-object-storage-interface-controller/issues/20